betatest/admin.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
```

Comment:
@antoviaque My main intention in the last comment was to point out that even the FSF recommendation to include the header everywhere isn't that strongly worded.  I didn't mean to imply any critique of the goals of the FSF in general, which I definitely agree with.